php/libraries/Utility.class.inc

```diff
@@ -132,12 +132,15 @@ class Utility
         $DB      = $factory->database();
 
         // get the list of study sites - to be replaced by the Site object
-        $query = "SELECT CenterID, Name FROM psc ";
+        $query  = "SELECT CenterID, Name FROM psc ";
+        $query .= "WHERE ";
         if ($study_site) {
-            $query .= "WHERE Study_site='Y'";
-        }
-        if (!$DCC) {
-            $query .= " AND CenterID <> 1";
+            $query .= "Study_site='Y' ";
+            if (!$DCC) {
+                $query .= " AND CenterID <> 1";
+            }
+        } else if (!$DCC) {
+            $query .= " CenterID <> 1";
```

Comment:
you created code redundancy here that is un-necessary. these lines do not need to be modified for this bugfix

Reply:
If both variables are set to true there needs to be an `AND` operator in between while for `Study_site=false` and `dcc=true` the AND operator would break the SQL query as it would become: 

```
SELECT CenterID, Name FROM psc     
WHERE AND CenterID <> 1
```

Reply:
that's correct but doesn't necessarily mean you need to duplicate the code.

try something else.